files/.pryrc

```diff
@@ -0,0 +1,8 @@
+Pry.config.print = proc do |output, value, _pry_|
+  case value
+  when ActiveRecord::Relation
+    output.puts "=> #{value.to_s}"
+  else
+    Pry::DEFAULT_PRINT.call(output, value, _pry_)
+  end
+end
```

Comment:
Let's fix this little line-ending violation before we put it into the template?

Reply:
🤦🏾 I don't know what my style guides are checking, but obviously not my code!